fixtures-generator/fixtures_faker/fixtures_faker.py

```diff
@@ -178,34 +178,34 @@ def create_shops(self, list_size=1):
             'shops': result
         }
 
-    def __category(self, pk, name):
+    def __image(self, pk, image_folder):
```

Comment:
why rename `category` to `image` ?

Reply:
I haven't renamed `category` to `image`. I have renamed `productimage` to `image` and moved code around.